libs/expo/betterangels/src/lib/screens/NotesHmis/HmisProgramNoteView/HmisProgramNoteTitle.tsx

```diff
@@ -0,0 +1,35 @@
+import { TextBold, TextRegular } from '@monorepo/expo/shared/ui-components';
+import { format } from 'date-fns';
+import { View } from 'react-native';
+import { HmisNoteQuery } from './__generated__/HmisProgramNoteView.generated';
+
+export default function HmisProgramNoteTitle({
+  hmisNote,
+}: {
+  hmisNote: HmisNoteQuery['hmisNote'] | undefined;
+}) {
+  if (hmisNote?.__typename !== 'HmisNoteType') return null;
```

Comment:
Is there a way for us to have a pattern where we only pass valid notes to the function vs having it check if we passed something valid?